source/features/show-associated-branch-prs-on-fork.tsx

```diff
@@ -42,11 +45,11 @@ const getPullRequestsAssociatedWithBranch = cache.function(async (): Promise<Rec
 
 	const pullRequests: Record<string, PullRequest> = {};
 	for (const {name, associatedPullRequests} of repository.refs.nodes) {
-		const [prInfo] = associatedPullRequests.nodes;
+		const [prInfo] = associatedPullRequests.nodes as PullRequest[];
 		// Check if the ref was deleted, since the result includes pr's that are not in fact related to this branch but rather to the branch name.
 		const headRefWasDeleted = prInfo?.timelineItems.nodes[0]?.__typename === 'HeadRefDeletedEvent';
 		if (prInfo && !headRefWasDeleted) {
-			prInfo.state = prInfo.isDraft && prInfo.state === 'OPEN' ? 'Draft' : upperCaseFirst(prInfo.state);
+			prInfo.state = prInfo.isDraft && upperCaseFirst(prInfo.state) === 'Open' ? 'Draft' : upperCaseFirst(prInfo.state) as keyof typeof stateClass;
```

Comment:
Is there another way to do this without adding the extra function?

Reply:
I don't understand why add the function in the first place, it's just changing the case

Reply:
Since 'OPEN' is not `keyof typeof stateClass` only 'Open'

Reply:
That string is only used for comparison, it’s not meant to be a specific type. The condition could have been 1===1 and it should have still passed. 

 The rule is wrong here and probably its confusion can be solved with parens in the right place